docs/okp4-cognitarium.md

```diff
@@ -13,13 +13,13 @@ Instantiate message
 |parameter|description|
 |----------|-----------|
 |`limits`|**[StoreLimitsInput](#storelimitsinput)**. Limitations regarding store usage.|
-|`limits.max_byte_size`|**[Uint128](#uint128)**. The maximum number of bytes the store can contains. The size of a triple is counted as the sum of the size of its subject, predicate and object, including the size of data types and language tags if any. Default to [Uint128::MAX] if not set, which can be considered as no limit.<br />**Default:** `"340282366920938463463374607431768211455"`|
```

Comment:
See [msg.rs#L272](https://github.com/okp4/contracts/blob/b7bc252e3e493dd7d6218bf7373d4454560febb2/contracts/okp4-cognitarium/src/msg.rs#L272) as the description for the `limits.max_byte_size` attribute.